cirq/ops/common_gates.py

```diff
@@ -454,11 +459,16 @@ def controlled(self,
         a generic qudit) and where the control is satisfied by the qubit being
         ON, as opposed to OFF.
 
+        Note that this only transforms into a CZPowGate (or controlled version
+        of that gate) if the global shift of the ZPowGate is 0, otherwise
+        it produces a normal ControlledGate.
+
```

Comment:
Looks like the above two paragraphs can actually be combined more concisely into a single paragraph listing the preconditions for the specialization to occur.

Reply:
Combined for this and all other gates.

Reply:
I was suggesting more along the lines of:
```
This method will specialize the control only if:
1. global shift is 0
2. the last of the controls is a default-type control qubit (followed by the clarifying what that means)
```
It would make the prerequisites clearer to the reader. But it's a minor documentation nit and not worth blocking the PR.